### DISCUSSION TITLE
Old Flypad

### DISCUSSION THREAD

**Discussion by Reisener** [flybywiresim/aircraft#5703] [Help] [answered]
Hi there,
I look at new FBW information every day.
All videos always explain the new Flypad with the menu on the left. I always have the current version of FBW either Stable or DEV. No matter what I update is only the flypad with the menu on top. After an update it can also be that the flypad is only white and I can't see anything. Can someone help me, I don't know what I did wrong at first.
thanks
DLH874 Bernd

**Comment by CapJambo** [comment, marked as answer]
I had same problem. You probably have 2 versions installed. You need to DELETE the one inside the OFFICIAL folder. I would recomende deleting the one inside the COMMUNITY folder too and reinstall from scratch.

Cheers

**Comment by Reisener** [comment]
Hello CapJambo,
a big thank you for this tip.
I did everything as you said. I had read somewhere that it doesn't work with the V 0.6.3 stable version. Then I loaded the DEV version and the new FlyPad was there and I can finally do the flights with the FBW.
Thanks again for the tip.
DLH874 Bernd

**Reply by CapJambo** [reply]
> 
> 
> Hello CapJambo,
> a big thank you for this tip.
> I did everything as you said. I had read somewhere that it doesn't work with the V 0.6.3 stable version. Then I loaded the DEV version and the new FlyPad was there and I can finally do the flights with the FBW.
> Thanks again for the tip.
> DLH874 Bernd

No problem. Came here to get help, and gave some!
Cheers
